src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -365,11 +365,11 @@ def _flexible_server_params(command_group):
         )
 
         mysql_high_availability_arg_type = CLIArgumentType(
-            arg_type=get_enum_type(['ZoneRedundant', 'SameZone', 'Disabled']),
+            arg_type=get_enum_type(['ZoneRedundant', 'SameZone', 'Disabled', 'Enabled']),
             options_list=['--high-availability'],
             help='Enable (ZoneRedundant or SameZone) or disable high availability feature. '
                  'Default value is Disabled. High availability can only be set during flexible server create time. '
-                 'Allowed values: ZoneRedundant, SameZone, Disabled'
+                 'Allowed values: ZoneRedundant, SameZone, Disabled, Enabled'
```

Comment:
CLI core will generate `Allowed values` automatically for enum type arguments, you can omit this line.

Reply:
Aha. thanks!